client/components/Slide/SlideEditor/index.jsx

```diff
@@ -194,51 +244,69 @@ export default class SlideEditor extends React.Component {
+                                                    }
+                                                },
+                                                delete: {
+                                                    onConfirm: () => {
+                                                        this.onDeleteComponent(
+                                                            index
+                                                        );
+                                                    }
                                                 }
+                                            }}
+                                        />
+                                    );
+                                    const display = <Display {...value} />;
+                                    return this.state.mode === 'edit' ? (
+                                        <Segment
+                                            key={index}
+                                            className={
+                                                index ===
+                                                this.state.currentComponentIndex
+                                                    ? 'editor__component-selected'
+                                                    : ''
                                             }
-                                        }}
-                                    />
-                                );
-                                const display = <Display {...value} />;
-
-                                return this.state.mode === 'edit' ? (
-                                    <Segment key={index}>
-                                        {componentMenu}
-                                        {edit}
-                                    </Segment>
-                                ) : (
-                                    <React.Fragment key={index}>
-                                        {display}
-                                    </React.Fragment>
-                                );
-                            })}
+                                            onClick={() =>
+                                                this.setState({
+                                                    currentComponentIndex: index
+                                                })
+                                            }
+                                        >
+                                            {componentMenu}
+                                            {edit}
+                                        </Segment>
+                                    ) : (
+                                        <React.Fragment key={index}>
+                                            {display}
+                                        </React.Fragment>
+                                    );
+                                })}
+                            </Sortable>
```

Comment:
This is just a bunch of whitespace changes caused by nesting the component render in `<Sortable>`